apps/mark-scan/pat-device-input/src/pin.rs

```diff
@@ -53,6 +53,16 @@ impl Pin {
             .expect("Unexpected failure to tear down pin");
     }
 
+    /// Attempts to export the pin. If successful, cleans up by unexporting
+    /// the pin and returning an empty Result. Returns an error if the pin
+    /// does not exist, the program lacks permission to export, or an
+    /// unexpected error occurs.
```

Comment:
Sort of self-explanatory but I wanted to enumerate the expected vs unexpected errors